### ISSUE TITLE
DB Lock are not removed

### ISSUE THREAD

**Issue by blink38** [fusioninventory/fusioninventory-for-glpi#1750]
In file inc/inventorycomputerinventory.class.php, in import() function, clean all db lock line are not executed :

```php
      // Clean all DB LOCK if exist more than 10 minutes
      $time = 600;
      $query = "DELETE FROM `glpi_plugin_fusioninventory_dblockinventorynames` "
              . " WHERE `date` <  CURRENT_TIMESTAMP() - ".$time;
      $query = "DELETE FROM `glpi_plugin_fusioninventory_dblockinventories` "
              . " WHERE `date` <  CURRENT_TIMESTAMP() - ".$time;
      $query = "DELETE FROM `glpi_plugin_fusioninventory_dblocksoftwares` "
              . " WHERE `date` <  CURRENT_TIMESTAMP() - ".$time;
      $query = "DELETE FROM `glpi_plugin_fusioninventory_dblocksoftwareversions` "
              . " WHERE `date` <  CURRENT_TIMESTAMP() - ".$time;

      // DB LOCK
```
I think it is missing some $DB->query() call in order to delete lock in tables.

In my case, I have some locks in table, avoiding some computer to update theirs informations. I also had to patch my code with this patch https://github.com/fusioninventory/fusioninventory-for-glpi/commit/b4182019cb303d4e02c2b6f33281b7de2cbc5af5 to not having apache slot busy for long time. This patch resolved my first problem, but the cause is because the locks are not deleted.

**Comment by ddurieux** [comment]
You're right

**Comment by ddurieux** [comment]
Fixed in 57ea39c
